app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -0,0 +1,192 @@
+module EmsPhysicalInfraHelper::TextualSummary
+  include TextualMixins::TextualRefreshStatus
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i(hostname ipaddress type port cpu_resources memory_resources cpus cpu_cores guid)
+    )
+  end
+
+  def textual_group_relationships
+    TextualGroup.new(
+      _("Relationships"),
+      %i(physical_infrastructure_folders folders clusters hosts datastores vms templates orchestration_stacks ems_cloud)
+    )
+  end
+
+  def textual_group_status
+    TextualGroup.new(
+      _("Status"),
+      textual_authentications(@record.authentication_userid_passwords) + %i(refresh_status orchestration_stacks_status)
+    )
+  end
+
+  def textual_group_smart_management
+    TextualTags.new(_("Smart Management"), %i(zone tags))
+  end
+
+  def textual_group_topology
+    items = %w(topology)
+    i = items.collect { |m| send("textual_#{m}") }.flatten.compact
```

Comment:
Sure?  Until there's more than one...

In reality, this should be a private method somewhere hiding the details of building this `"textual_"` prefixed items since there's already 17[1] other occurrences of this exact line.  And, if you want to optimize on whether there's only one item, then the private method would be a fantastic place to do that. 

```
$ git grep -Ii "items.collect { |m| send(\"textual_#{m}" 
app/helpers/container_build_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_group_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_image_helper/textual_summary.rb:      items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_image_registry_helper/textual_summary.rb:      items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_node_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_project_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_replicator_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_route_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_service_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/container_template_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/ems_cloud_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/ems_container_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/ems_infra_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/ems_middleware_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/ems_network_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
app/helpers/persistent_volume_helper/textual_summary.rb:    items.collect { |m| send("textual_#{m}") }.flatten.compact
```